tool/src/test/java/org/wildfly/security/tool/MaskCommandTest.java

```diff
@@ -63,9 +64,9 @@ public void testMissingSaltAndIteration() {
         String retVal = executeCommandAndCheckStatusAndGetOutput(args);
         String[] retValLines = retVal.split(LINE_SEPARATOR);
 
-        assertTrue("Message about invalid salt parameter must be present", retValLines[0].contains("Invalid \"salt\" parameter. Generated value"));
-        assertTrue("Message about invalid iteration parameter must be present", ("Invalid \"iteration\" parameter. Default value \"" + defaultIteration + "\" will be used.").equals(retValLines[1]));
-        assertTrue("Message about invalid salt parameter must be present", retValLines[2].contains("MASK-"));
+        assertEquals("Message about invalid salt parameter must be present", true, retValLines[0].contains("Invalid \"salt\" parameter. Generated value"));
```

Comment:
i think we can keep the original statement that asserts `retValLines[0].contains("Invalid \"salt\" parameter. Generated value")` is true, rather than asserting that statement is equal to true